Packages/Testing-MIES/UTF_BasicHardwareTests.ipf

```diff
@@ -2905,6 +2905,118 @@ Function TPDuringDAQWithoodDAQ_REENTRY([str])
 	CHECK_EQUAL_TEXTWAVES(stimsets, {"TestPulse", "StimulusSetC_DA_0", "StimulusSetC_DA_0", "", "", "", "", "", ""}, mode = WAVE_DATA)
 End
 
+Function TPDuringDAQTPStoreCheck_IGNORE(device)
+	string device
+
+	PGC_SetAndActivateControl(device, "check_Settings_RequireAmpConn", val = 0)
+
+	PGC_SetAndActivateControl(device, "check_Settings_TP_SaveTP", val = 1)
+
+	PGC_SetAndActivateControl(device, GetPanelControl(0, CHANNEL_TYPE_HEADSTAGE, CHANNEL_CONTROL_CHECK), val = 1)
+	PGC_SetAndActivateControl(device, GetPanelControl(0, CHANNEL_TYPE_DAC, CHANNEL_CONTROL_WAVE), str = "TestPulse")
+
+	PGC_SetAndActivateControl(device, GetPanelControl(1, CHANNEL_TYPE_HEADSTAGE, CHANNEL_CONTROL_CHECK), val = 1)
+	PGC_SetAndActivateControl(device, GetPanelControl(1, CHANNEL_TYPE_DAC, CHANNEL_CONTROL_WAVE), str = "PulseTrain_10Hz_DA_0")
+
+	PGC_SetAndActivateControl(device, GetPanelControl(2, CHANNEL_TYPE_HEADSTAGE, CHANNEL_CONTROL_CHECK), val = 1)
+	PGC_SetAndActivateControl(device, GetPanelControl(2, CHANNEL_TYPE_DAC, CHANNEL_CONTROL_WAVE), str = "TestPulse")
+End
+
+Function TPWaitForAsync_IGNORE(TPStorage, storedTP, timeOut)
+	WAVE TPStorage
+	WAVE storedTP
+	variable timeOut
+
+	variable size1, size2, endTime
+	string errorMsg, empty
+
```

Comment:
How about hard coding the timeout? Otherwise you need to also check that it is positive and reasonably selected.